common/models/preview-batch.json~

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
It looks like there are two files, the canonical `preview-batch.json` and a backup file `preview-batch.json~` (with the tilde). Please delete the backup file.

It helps to edit [.gitignore](https://github.com/civicrm/prevem/blob/master/.gitignore) and add a line `*~`. This will prevent us from adding more backup/tilde files in the future.